algorithms/sorting.md

```diff
@@ -57,11 +57,17 @@ Say we want to sort a deck of cards:
 4. If not, repeat lines 1 - 3
 
 Let's talk about the Complexity of BogoSort. Here's the pseudocode:
```

Comment:
The code that you added is not called as pseducode.

Reply:
I updated the pseudo-code. can you verify now?